addons/zylann.scatter/tools/palette.tscn

```diff
@@ -71,6 +71,80 @@ max_value = 10.0
 step = 0.01
 allow_greater = true
 
+[node name="SizeContainer" type="HBoxContainer" parent="VBoxContainer"]
+margin_top = 512.0
+margin_right = 202.0
+margin_bottom = 536.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="MinSizeLabel" type="Label" parent="VBoxContainer/SizeContainer"]
+margin_top = 5.0
+margin_right = 99.0
+margin_bottom = 19.0
+size_flags_horizontal = 3
+text = "Min size (%)"
+align = 1
+
+[node name="MinSizeValue" type="LineEdit" parent="VBoxContainer/SizeContainer"]
```

Comment:
Don't use LineEdit when the input is supposed to be a number. SpinBox is better, but I think a single slider would be even simpler

Reply:
Perhaps it would be easier to use just one input, but it would not have been that good for "my" use-case (I wanted variation of 50-110% which would be quite un-intuitive to enter with just one slider).

Will try to realize this with spin-boxes and implement some plausibility checks (min<max, and so on)